docs(changelog): changelog 2023-01-11 (#1193)
scaleway/docs-content: changelog/serverless/january2023/2023-01-11-fixed-functions-doublecorserror.mdx

---
title: Fixed the double CORS error on Functions!
status: fixed
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-01-11
category: serverless
product: functions
---

> The double CORS error that may have occured on Serveless Functions has been fixed.

Reviewer: ```suggestion
The double CORS error that occasionally occurred on Serveless Functions has been fixed.
```
